### DISCUSSION TITLE
Garage Band Setup thread

### DISCUSSION THREAD

**Discussion by quantuminformation** [ExistentialAudio/BlackHole#44] [General]
I got this to work in GarageBand the following settings also allow me to record with the external mike

Channel 17 for Mac internal mic
![image](https://user-images.githubusercontent.com/216566/72020630-15ad0380-3264-11ea-91c3-62d13b6906ff.png)

And channels one and two for computer audio

![image](https://user-images.githubusercontent.com/216566/72020648-23628900-3264-11ea-891a-40952fcab0d3.png)

Aggregate device
![image](https://user-images.githubusercontent.com/216566/72020673-2f4e4b00-3264-11ea-9ad4-e8682bd3cc3a.png)

And multi output
![image](https://user-images.githubusercontent.com/216566/72020689-39704980-3264-11ea-86b3-ea2879dcc514.png)

Nice work so far!

**Comment by devinroth** [comment]
Awesome. Thanks.

**Comment by odinian** [comment]
Here's my instructions for how to set up GarageBand to use it (with BlackHole) for system wide EQ or any other AU plugin.
https://sites.google.com/marcusfamily.info/diynotes/tips/mac-eq-plugins/garageband
Hopefully someone finds it useful.

**Comment by ronkorving** [comment]
@odinian That URL gives me a 404

**Comment by odinian** [comment]
@ronkorving oops.
https://sites.google.com/marcusfamily.info/diynotes/tips/mac-eq-plugins/garageband
